packer/pmm2.pkr.hcl

```diff
@@ -0,0 +1,192 @@
+
+variable "pmm2_server_repo" {
+  type    = string
+  default = "testing"
+}
+
+variable "pmm_client_repo_name" {
+  type    = string
+  default = "percona-testing-x86_64"
```

Comment:
Is it a real repository name? I couldn't find it )

Reply:
Yep!
It's testing repo:
```
[root@459f63962611 opt]# grep -r percona-testing-x86_64 /etc/yum.repos.d/
/etc/yum.repos.d/percona-original-testing.repo.bak:[percona-testing-x86_64]
```